src/FluffySpoon.AspNet.LetsEncrypt.Azure/AzureAppServiceSslBindingCertificatePersistenceStrategy.cs

```diff
@@ -69,7 +69,7 @@ public async Task PersistAsync(PersistenceType persistenceType, byte[] bytes)
 
 			var domains = letsEncryptOptions.Domains.ToArray();
 
-			logger.LogInformation("Creating new Azure certificate for key {0} and domains {1}.", persistenceType, domains);
```

Comment:
Why did you make this change? Isn't structured logging data better? That way, you can store it in structured form in ELK or similar.

Reply:
It used to be like that, but it isn't anymore.

https://docs.microsoft.com/en-us/aspnet/core/fundamentals/logging/?view=aspnetcore-3.0#log-message-template

Search for "structured" on that page.

It's up to the log provider to decide what to do with the data. The console provider will just stringify it though.

Reply:
Reverted.